From 421886d7d5bbbea6277f6a5fbd3e7cc685bf365e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fran=C3=A7ois=20Revol?= <revol@free.fr>
Date: Mon, 17 Feb 2025 14:40:18 +0100
Subject: [PATCH] Haiku: Add globbing for SCSI disks

On machines without ATA emulation disks appear as SCSI with bus/dev/lun:
/dev/disk/scsi/0/0/0/raw
---
 src/hdaccess.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/hdaccess.c b/src/hdaccess.c
index 9768c07c..aefb125b 100644
--- a/src/hdaccess.c
+++ b/src/hdaccess.c
@@ -462,6 +462,7 @@ list_disk_t *hd_parse(list_disk_t *list_disk, const int verbose, const int testd
   {
 #ifdef HAVE_GLOB_H
     list_disk=hd_glob_parse("/dev/disk/*/*/*/raw", list_disk, verbose, testdisk_mode);
+    list_disk=hd_glob_parse("/dev/disk/*/*/*/*/raw", list_disk, verbose, testdisk_mode);
 #endif
   }
 #else
-- 
2.45.2

